### ISSUE TITLE
Upgrading to v4.15.0 breaks (@babel/runtime again)

### ISSUE THREAD

**Issue by ddolcimascolo** [mjmlio/mjml#2819]
**Describe the bug**

Upgrading to v4.15.0 breaks with

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './helpers/callSuper' is not defined by "exports" in /data/dev/mjml-babel-4.15.0/node_modules/@babel/runtime/package.json
    at new NodeError (node:internal/errors:406:5)
    at exportsNotFound (node:internal/modules/esm/resolve:268:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:598:9)
    at resolveExports (node:internal/modules/cjs/loader:547:36)
    at Module._findPath (node:internal/modules/cjs/loader:621:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1034:27)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (/data/dev/mjml-babel-4.15.0/node_modules/mjml-core/lib/index.js:71:42) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
```

**To Reproduce**
1. mkdir mjml-babel-4.15.0
2. cd mjml-babel-4.15.0
3. npm init -y
4. npm i --save -E mjml@4.14.1 @babel/runtime@7.14.6 (to for the @babel/runtime dependency to the _existing_ version, and the version mjml depends on)
5. npm ls @babel/runtime (to verify the installed version)
6. node -e 'require("mjml")' (expect no error)
7. npm i mjml@latest
8. npm ls @babel/runtime (version 7.14.6 is **still installed**)
9. node -e 'require("mjml")' (:boom:)

**Expected behavior**

Upgrade works

**MJML environment (please complete the following information):**
 - OS: [e.g. Linux]
 - MJML Version [e.g. 4.15.0]
 - MJML tool used [e.g MJML Api]

**Email sending environment(for rendering issues)**:
 
N/A

**Affected email clients (for rendering issues):**

N/A

**Screenshots**

**Additional context**

**Comment by iRyusa** [comment]
Not sure why you add manually `@babel/runtime@7.14.6` to your package json here ?

`npm prune` should just fix this no?

**Comment by ddolcimascolo** [comment]
Hi @iRyusa 

I do it to simulate an existing project having the previous version installed. This problem is not new (https://github.com/mjmlio/mjml/issues/2299) and is due to lax dependencies version... I'm using mjml for a very long time, so my `@babel/runtime` dependency is whatever version satisfied your package.json at that time, actually 7.14.6. MJML now requires a more recent version of `@babel/runtime` but you didn't bump the dependency in package.json, resulting in a broken package for upgrades.

Indeed the problem does not appear for _new_ installs in new projects because NPM picks the latest available version of `@babel/runtime` that satisfies the dependency.

According to Babel, you should depend on at least 7.23.8 as per https://github.com/babel/babel/blob/0e5ae8de66d5c1e9ecc3f94daa5bd5d3a920e5a4/packages/babel-helpers/src/helpers/callSuper.ts#L1

Hope this helps,
David

**Comment by iRyusa** [comment]
Really need to burn everything down and remove completely babel at some point 😮‍💨 .

I'll do a 4.15.2 with updated package.json for everything for today

**Comment by iRyusa** [comment]
For future me : `yarn upgrade-interactive` doesn't update the `package.json` when updating a `package` so I need to do update manually next time 😑 

Can you confirm 4.15.2 is good for you @ddolcimascolo ?

**Comment by ddolcimascolo** [comment]
Yep. All good now.

Thanks again,
Cheers,

David

**Comment by ddolcimascolo** [comment]
Fixed in 4.15.2
